plugin/pkg/scheduler/backoff_utils.go

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
I'm slightly against putting this file into main directory. How about:
plugin/pkg/scheduler/utils/backoff.go
?